start

```diff
@@ -377,31 +375,49 @@ function init_horizon() {
   fi
   pushd $HZHOME
 
-  run_silent "chown-horizon" chown stellar:stellar .
-
   sed -ri \
     -e "s/__PGPASS__/$PGPASS/g" \
     -e "s/__NETWORK__/$NETWORK_PASSPHRASE/g" \
     -e "s=__ARCHIVE__=$HISTORY_ARCHIVE_URLS=g" \
     etc/horizon.env
 
   if [ "$ENABLE_HORIZON_CAPTIVE_CORE" = "true" ]; then
+    
+    HORIZON_CAPTIVE_CORE_CFG=$COREHOME/etc/horizon-captive-core.cfg
+    cp $COREHOME/etc/stellar-captive-core.cfg $HORIZON_CAPTIVE_CORE_CFG 2>/dev/null || true
+    run_silent "finalize-horizon-captivecore-db" sed -ri "s*__DATABASE__*sqlite3://$COREHOME/stellar.db*g" $HORIZON_CAPTIVE_CORE_CFG
+    run_silent "finalize-horizon-captivecore-peer-port" sed -ri "s*__PEER_PORT__*11725*g" $HORIZON_CAPTIVE_CORE_CFG
+    sed -ri "s/__NETWORK__/$NETWORK_PASSPHRASE/g" $HORIZON_CAPTIVE_CORE_CFG
+
+    mkdir ./cc
```

Comment:
Can we give this directory a self describing name? Maybe `captivecore`? It took me a while digging around to realize what this folder was for.

Reply:
renamed, https://github.com/stellar/quickstart/pull/414/commits/fda507ed2836f2efc01e7426301b7b2f64f2f8bb